src/shared/defineFeatured.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable max-len */
+import type {SIM} from './browserCaching'
+
+// Use the array below to define the featured specimens.
+// Note that `en64` is the base64 encoding of the desired specimens, which
+// can be found as the value of the `specimen=` query string in the URL.
+
+export function getFeatured(): SIM[] {
+    const featuredSIMs = [
+        {
+            en64: 'eyJuYW1lIjoiQmFzaWMgTW9kZmlsbCIsInNlcXVlbmNlIjoiUmFuZG9tIiwic2VxdWVuY2VQYXJhbXMiOiJleUp0YVc0aU9pSXlJaXdpYldGNElqb2lNVFFpZlE9PSIsInZpc3VhbGl6ZXIiOiJNb2RGaWxsIiwidmlzdWFsaXplclBhcmFtcyI6ImV5SnRiMlJFYVcxbGJuTnBiMjRpT2lJeE1pSjkifQ==',
+            name: 'Basic Modfill',
```

Comment:
It is a source of non-DRYness that the name is defined at all here, instead of taken straight out of the en64 variable, in which a name is already encoded, right?  I noticed this because I was suggesting some to include and had a name mismatch.  That shouldn't be able to occur, right?

Reply:
See [this suggestion](https://github.com/orgs/numberscope/discussions/6#discussioncomment-9926868). I was going to leave that for post-all-current-PRs, but if it's a direction we want to go, we could tackle it now and address this point. I agree, it does seem very awkward to have redundant information that can be internally out of sync, and would be happy to clear that up one way or another.

Reply:
This redundancy bugs me, but I'm on the fence about whether to deal with it now, because of the human-readable question.  Does anyone else have a preference?

Reply:
I would definitely like to explore human-readable URLs at some point because I find the base64 ones fairly offputting. I am on the fence as to whether to address that now or after all PRs. I guess my feeling is that the point at which we try out human-readable PRs is the appropriate time to address the redundancy. But I am assuming this PR will not be merged before tomorrow's meeting, and so we should definitely discuss all outstanding points about this PR and when to address them, at the meeting.

Reply:
The question of how to enable human-readable URLs seems subtle to me. I vote for trying to do this after all the CSE 2000 PRs.

Reply:
Have removed the explicit redundancy in defineFeatured.ts, and human-readable URLs has been deferred to a later issue. Resolving.